larpandoracontent/LArThreeDReco/LArHitCreation/ThreeDHitCreationAlgorithm.cc

```diff
@@ -286,13 +507,14 @@ void ThreeDHitCreationAlgorithm::RefineHitPositions(const ThreeDSlidingFitResult
         }
         else
         {
-            std::cout << "ThreeDHitCreationAlgorithm::IterativeTreatment - Unexpected number of trajectory samples" << std::endl;
-            throw StatusCodeException(STATUS_CODE_FAILURE);
+            // std::cout << "ThreeDHitCreationAlgorithm::IterativeTreatment - Unexpected number of trajectory samples" << std::endl;
+            // throw StatusCodeException(STATUS_CODE_FAILURE);
+            continue; // TODO: Fix this, check why it happens and if it is specific to the changes I've made.
```

Comment:
Looking into, as mentioned in the TODO...no idea if seeing this warning is my fault (I can't see how, I always give a trajectory sample). But it was getting in the way of producing metrics consistently by crashing out too early, so I dropped the error for now. I'll verify if its my doing, or just an oddity I was seeing with empty events that should be expected.